source/MaterialXView/Glsl/GLUtil/WindowWrapper.cpp

```diff
@@ -0,0 +1,219 @@
+#include <MaterialXView/Glsl/GLUtil/WindowWrapper.h>
```

Comment:
A "window" is composed of various platform dependent handles. This is the wrapper to encapsulate this as much as possible.